src/main/scala/formal/FormalUtils.scala

```diff
@@ -5,6 +5,7 @@ import Chisel._
 import chisel3.{VecInit}
 import chisel3.util.Cat
 import chisel3.internal.sourceinfo.{SourceInfo, SourceLine}
+import freechips.rocketchip.config._
```

Comment:
```suggestion
import freechips.rocketchip.config.Field
```
Better to not wildcard import